### ISSUE TITLE
A few issues updating pack from 1.30 to 1.31

### ISSUE THREAD

**Issue by Sandman366** [NillerMedDild/Enigmatica2ExpertSkyblock#529]
**Modpack version**
v1.31

**Issue**
Okay, first off, Wireless Crafting Terminal have some broken textures in the versions for this version of the pack. Tried using anyway to see if it was just the item texture....nope, crash on use. So, update that to fix broken textures....nope, didn't realize the dependencies also required an update. Okay, _now_ see if it'll actually start again....

Nope! Dank/null wants to crash, now. I'll be honest here, didn't care enough to update/try to fix this one, just disabled. (Haven't used it before and don't expect to use it anyway, just wanted to get moving on again.)

Okay, pack doesn't crash while starting up again (yay!), can use WCT again (yay!), but _now_ Modular Machinery machines and recipes are entirely broken. Machine blueprints are craftable, but they all say "empty", blueprints have no uses in JEI, and already-built ones say unknown structure/blueprint. Cause appears to be a section in config/modularmachinery/machinery/variables/casings.var.json about mana hatches for Botania casings. Changelog says they were removed, so tried deleting that section in the config. Machines work again (yay!), but this line has to be removed from both client _and_ server configs, and (server files at least) it's in the config that comes in v1.31.

tl;dr:
1. p455w0rdslib, WCT, and AE2WTLib all require updates to work again. As bundled in current pack version, WCT will crash on use, and it and other items have broken item textures. (Versions 2.2.156, 3.12.96, and 1.0.32, respectively, worked for me.)
2. dank/null doesn't work at this point (crash during startup). Update might fix, I didn't care enough and just dropped it.
3. Modular Machinery needs casings_botania section deleted from casings.var.json, or no modular machines work. (Client _and_ server configs!)

Edit: Oh goody, apparently FTBUtilities also now has a crash with picking up EnderIO machines with a crescent hammer, and isn't fixed yet....but don't know how far back to go for it to not be a problem either. :(
Edit2: Tried a couple of recent versions to downgrade and didn't fix, didn't want to individually go back versions to find a fix, so just jumped back to versions of FTBLib and FTBUtils from July 1st, and that particular crash is gone now, at least.

**Comment by NillerMedDild** [comment]
I'll hotfix today, thanks :)
